client/daemon/storage/storage_manager.go

```diff
@@ -716,6 +716,9 @@ func (s *storageManager) forceGC() (bool, error) {
 }
 
 func (s *storageManager) diskUsageExceed() (exceed bool, bytes int64) {
+	if s.storeOption.DiskGCThresholdPercent <= 0 {
```

Comment:
if DiskGCThresholdPercent <= 0, diskUsageExceed never return true?

Reply:
DiskGCThresholdPercent is always great than 0 or equal zero